src/Test.UnitTests.Sarif.Driver/Sdk/AnalyzeCommandBaseTests.cs

```diff
@@ -1405,26 +1430,26 @@ public void AnalyzeCommandBase_MultithreadedShouldUseCacheIfFilesAreTheSame()
             // of files tend to have the same name but appear in different directories). For a source code scanner,
             // however, two files in the same directory may hash the same (an empty file that produces no scan
             // results is an obvious case).
-            $@"{Environment.CurrentDirectory}\Error.1.of.5.cpp",
-            $@"{Environment.CurrentDirectory}\Error.2.of.5.cs",
-            $@"{Environment.CurrentDirectory}\Error.3.of.5.exe",
-            $@"{Environment.CurrentDirectory}\Error.4.of.5.h",
-            $@"{Environment.CurrentDirectory}\Error.5.of.5.sys",
-            $@"{Environment.CurrentDirectory}\Warning.1.of.2.java",
-            $@"{Environment.CurrentDirectory}\Warning.2.of.2.cs",
-            $@"{Environment.CurrentDirectory}\Note.1.of.3.dll",
-            $@"{Environment.CurrentDirectory}\Note.2.of.3.exe",
-            $@"{Environment.CurrentDirectory}\Note.3.of.3jar",
-            $@"{Environment.CurrentDirectory}\Pass.1.of.4.cs",
-            $@"{Environment.CurrentDirectory}\Pass.2.of.4.cpp",
-            $@"{Environment.CurrentDirectory}\Pass.3.of.4.exe",
-            $@"{Environment.CurrentDirectory}\Pass.4.of.4.dll",
-            $@"{Environment.CurrentDirectory}\NotApplicable.1.of.2.js",
-            $@"{Environment.CurrentDirectory}\NotApplicable.2.of.2.exe",
-            $@"{Environment.CurrentDirectory}\Informational.1.of.1.sys",
-            $@"{Environment.CurrentDirectory}\Open.1.of.1.cab",
-            $@"{Environment.CurrentDirectory}\Review.1.of.2.txt",
-            $@"{Environment.CurrentDirectory}\Review.2.of.2.dll"
+            $@"{Environment.CurrentDirectory}{Path.DirectorySeparatorChar}Error.1.of.5.cpp",
```

Comment:
>$@"{Environment.CurrentDirectory}{Path.DirectorySeparatorChar} [](http://example.com/codeflow?start=12&length=62)

const string rootDir = $@"{Environment.CurrentDirectory}{Path.DirectorySeparatorChar";

$@"{RootDir}Error.1.of.5.cpp

Reply:
fixed